src/components/core/Root/index.tsx

```diff
@@ -34,7 +34,11 @@ const FusionRoot: React.FC<FusionRootProps> = ({
                 ref={overlayRef as React.MutableRefObject<HTMLDivElement>}
             />
             <div className={styles.snacks}>
-                <NotificationSnacks registerPresenter={notificationCenter.registerPresenter.bind(notificationCenter)} />
+                <NotificationSnacks
+                    registerPresenter={notificationCenter.registerPresenter.bind(

```

Comment:
Instead of binding the registerPresenter method, you could use registerPresenter = () => {} on the notificationCenter